packages/teleterm/src/ui/commandLauncher.test.ts

```diff
@@ -0,0 +1,36 @@
+import { MockAppContext } from 'teleterm/ui/fixtures/mocks';
```

Comment:
Missing license here?

Reply:
I haven't been adding license to any file since I've joined. 🙊

If that's necessary then once the merge is done we should just modify the ignored paths of `make fix-license`.